app/src/main/java/org/mozilla/fenix/tabtray/TabTrayInteractor.kt

```diff
@@ -0,0 +1,18 @@
+package org.mozilla.fenix.tabtray
+
+@SuppressWarnings("TooManyFunctions")
+class TabTrayInteractor(
```

Comment:
Can we add a doc to this similar to https://github.com/mozilla-mobile/fenix/blob/master/app/src/main/java/org/mozilla/fenix/library/history/HistoryInteractor.kt#L7